app/src/main/java/yt/javi/quotes/quotesapp/MainActivity.kt

```diff
@@ -1,19 +1,20 @@
 package yt.javi.quotes.quotesapp
 
-import android.app.ListActivity
+import android.app.Activity
 import android.os.Bundle
+import android.view.View
 import android.widget.ArrayAdapter
+import android.widget.ListView
 
 
-class MainActivity : ListActivity() {
-    private lateinit var adapter: ArrayAdapter<String>
-
+class MainActivity : Activity() {
```

Comment:
It's ok to use activity, but is better to extend from AppCompatActivity as it will provide several support things to be able to use material in older versions

Reply:
Done :)